packages/botonic-nlp/tests/embeddings/embedder.test.ts

```diff
@@ -1,10 +1,9 @@
-import { DatabaseManager } from '../../src/embeddings/database/manager'
 import { Embedder } from '../../src/embeddings/embedder'
+import * as helper from '../helpers/tasks/ner/helper'
 
 describe('Embedder', () => {
   test('Generate embeddings matrix', async () => {
-    const manager = await DatabaseManager.with('en', 'glove', 50)
-    const embedder = new Embedder(manager)
+    const embedder = new Embedder(helper.testWordEmbeddingManager)
```

Comment:
rename Embedder to WordEmbedder

Reply:
Do you refer to the class Embedder?? If so, I think WordEmbedder might not be the best name because the only method it has is for creating the embeddings matrix, not for just getting the embedding of a word. What do you think? @dpinol